crate/operator/handlers/handle_update_cratedb.py

```diff
@@ -138,13 +140,16 @@ async def update_cratedb(
                     do_change_compute = True
                     # pod resources won't change until each pod is recreated
                     do_restart = True
+        elif field_path == ("spec", "backups", "aws", "cron"):
```

Comment:
I believe we discussed perhaps using a field handler (i.e. similar to CIDR changes) for this instead of adding more stuff to the update handler? We don't really need to do any of the other stuff that the update handler handles for cronjob changes (i.e. disabling backups).

Reply:
Changed it to use the field handler